todo/src/Mycomponent/Display.css

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
css and image file name should be in camelCase. So rename Display.css > display.css